src/classes/RD_RecurringDonations_TEST.cls

```diff
@@ -469,26 +469,354 @@ public class RD_RecurringDonations_TEST {
         integer cOpp =  [select count() from Opportunity where npe03__Recurring_Donation__r.Name = 'testweekly'];
         system.assert(cOpp == 52 || cOpp == 53);        
     }
-    
+
+    static testMethod void updateFixedLengthRecurringDonationIncrease(){
+        if (strTestOnly != '*' && strTestOnly != 'updateFixedLengthRecurringDonationIncrease') return;
```

Comment:
Now that we can run individual test methods directly through the IDE and DevConsole, this old pattern of `if (strTestOnly != '*' && strTestOnly != ....` is no longer necessary.